modules/ROOT/pages/visualizer-app-network.adoc

```diff
@@ -9,10 +9,10 @@ Anypoint Visualizer automatically generates the topology of the application.
 
 To maximize visualization readability, consumer nodes (nodes that have only outbound connections) are shown at the top. Backend systems (nodes that have only inbound connections) are shown at the bottom.
 
-When the `ClientId` is detected for a consumer node, the nodes are grouped by `ClientId`; otherwise, it is grouped under the *External Traffic* node. 
-The target `application/API/proxy` should have the `ClientId` enforcement policy to detect the `ClientId`. 
-For usability purposes, client app names replace `ClientIds` in the canvas. 
-All client apps that call the same Mule app are grouped into a single client app group node. 
+When the `ClientId` is detected for a consumer node, the nodes are grouped by `ClientId`; otherwise, it is grouped under the *External Traffic* node.
```

Comment:
```suggestion
When the `ClientId` category is detected for a consumer node, the nodes are grouped by `ClientId`; otherwise, they are grouped under the *External Traffic* node.
```

Reply:
Syntax values should not be used as common nouns, so we need to follow them with a common noun (typically "value" or "option") OR just use the common noun (in this case, "the client ID").
GLOBAL